raft/src/main/java/org/apache/kafka/raft/LeaderState.java

```diff
@@ -126,8 +127,13 @@ public long timeUntilCheckQuorumExpires(long currentTimeMs) {
      */
     public void updateCheckQuorumForFollowingVoter(int id, long currentTimeMs) {
         updateFetchedVoters(id);
-        // The majority number of the voters excluding the leader. Ex: 3 voters, the value will be 1
-        int majority = voterStates.size() / 2;
+        // The majority number of the voters. Ex: 3 voters, the value will be 2
+        int majority = (int) ((double) (voterStates.size() + 1) / 2);
```

Comment:
just curios. this function will return majority number `2` if there are 4 voters. Is that expected? I assume the corrected number is `3`

Reply:
Ah, good catch! My test didn't work as expected. Also updated the test to catch the issue. Thanks.